VenafiPS/Public/Test-TppToken.ps1

```diff
@@ -0,0 +1,116 @@
+<#
+.SYNOPSIS
+Test if a Tpp token is valid
+
+.DESCRIPTION
+Use the TPP API call 'Authorize/Verify' to test if the current token is valid.
+
+.PARAMETER AuthServer
+Auth server or url, venafi.company.com or https://venafi.company.com.
+If just the server name is provided, https:// will be appended.
+
+.PARAMETER AccessToken
+Access token retrieved outside this module.  Provide a credential object with the access token as the password.
+
+.INPUTS
+Accesstoken
+
+.OUTPUTS
+Boolean or PSCustomObject with the following properties:
+    AuthUrl
+    AccessToken
+    RefreshToken
+    Scope
+    Identity
+    TokenType
+    ClientId
+    Expires
+
+.EXAMPLE
+Test-TppToken -AuthServer 'mytppserver.example.com' -AccessToken $cred
+Verify that current accesstoken is valid
+
```

Comment:
need an example for venafisession parameter set

Reply:
Fixed